base/MAPL_NominalOrbitsMod.F90

```diff
@@ -625,7 +621,7 @@ SUBROUTINE find_sweeppoints(iii, latwayp, longwayp, l, r, wrapon, latlonshift)
                CALL linspace(r,l,iii,myvec)
        end if
        do i=1, size(myvec)
-          myvec_deg(i) =  ckm2deg * myvec(i)
+          myvec_deg(i) =  MAPL_KM2DEG * myvec(i)
```

Comment:
As long as we are giving a new name here,  I would like to avoid the "cute" use of "2" to represent English "to".   How about  `MAPL_DEG_PER_KM`?

Reply:
@Gvilla1000-nasa if you could do a sweep of such things.   (But only where we are already changing the name to have the MAPL prefix anyway.)

Reply:
Hmm.   Looking at other examples, I guess "PER" is not common.    So a backup would be  `MAPL_KM_TO_DEG` though I still think "PER" is clearer.

Reply:
Sounds good. I'll address in next push. I also noticed that I didn't remove `MAPL_UNDEF` from the Python Constants so I'll also take care of that.